### PR TITLE
SQL schema file rebase fix

### PR DESCRIPTION
**Pull request by Jibbscript**
Added lines to SQL schema and rebased Sean's updates to SQL schema

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -61,39 +61,60 @@ app.get('/menuitems', (req, res) => {
...
-        "query":`${req.query.searchInput}`
+        "query":req.query.searchInput
       }
     });
-    const recipeIDs = await recipesSearched.body.results.map(x => x.id);
+    
+    recipesData.results = await recipesSearched.body.results.map(x => {
+      delete x['readyInMinutes'];
+      delete x['imageUrls'];
+      return x;
```
Comment (juliakim) [inline server/server.js]: Definitely need at least 1 image URL to show with search results and prep time would also be nice to have. Consider removing lines 83-86, unless you are storing this data elsewhere.

File: server/server.js [hunk 2]
```diff
@@ -61,39 +61,60 @@ app.get('/menuitems', (req, res) => {
...
+    res.send(recipesData).status(200);
+    
+  } catch(err) {
+    console.log(err);
+  }
   
-
+});
+    
+
```
Comment (juliakim) [inline server/server.js]: Remove unnecessary line breaks

File: server/server.js [hunk 3]
```diff
@@ -61,39 +61,60 @@ app.get('/menuitems', (req, res) => {
...
+  } catch(err) {
+    console.log(err);
+  }
   
-
+});
+    
+    
+    
+    const recipeIDs = await recipesSearched.body.results.map(x => x.id);
```
Comment (juliakim) [inline server/server.js]: Change x to recipe or a more descriptive variable name.

File: server/server.js [hunk 4]
```diff
@@ -61,39 +61,60 @@ app.get('/menuitems', (req, res) => {
...
         },"params":{
           "ids":recipeIDs.join()
         }
     });
-    res.send(recipesInfoBulk).status(200);
+<<<<<<< HEAD
+=======
+    //recipesInfoBulk.body.map
+
+
```
Comment (juliakim) [inline server/server.js]: Remove please

File: server/server.js [hunk 5]
```diff
@@ -105,6 +126,7 @@ app.get('/searchRecipes', async (req, res) => {
     // })
 });
 
+>>>>>>> 508a5c9a7e4887983383053c1fbe4b5be7eb070a
```
Comment (juliakim) [inline server/server.js]: Delete this as well

**Review by juliakim — CHANGES_REQUESTED** [review]
Please fix commented issues.

**Comment by Jibbscript** [comment]
[Build Search endpoint for REST API](https://trello.com/c/OKFmHZgg/56-build-search-endpoint-for-rest-api)

**Review by SeanMcCarthy3223 — APPROVED** [review]

**Review by SeanMcCarthy3223 — APPROVED** [review]

**Pull request merged**